helmfile.d/values/fluentd/forwarder-common.yaml.gotmpl

```diff
@@ -233,6 +233,21 @@ extraConfigMaps:
       @type kubernetes_metadata
     </filter>
 
+    # count the number of incoming records per tag
```

Comment:
This seems a bit out-of-scope for this PR. Could it be extracted to its own PR?
Is this a nicely behaved metric or do we risk [cardinality explosion](https://grafana.com/blog/2022/02/15/what-are-cardinality-spikes-and-why-do-they-matter/)?

Reply:
I can move it into its own PR and it can be discussed there, I have not looked into the cardinality of the metric. But if we don't want the metric, if it misbehaves or is redundant than we should remove those fluentd panels instead, but that can be handled in a separate PR.